specification/containerservice/Fleet.Management/fleetnamespace.tsp

```diff
@@ -120,28 +121,241 @@ union EgressPolicy {
   string,
 }
 
-@doc("The profile of the fleet managed namespace.")
+@doc("The possible values representing different network policy rules.")
+union PolicyRule {
+  @doc("Allow all network traffic.")
+  AllowAll: "AllowAll",
+
+  @doc("Deny all network traffic.")
+  DenyAll: "DenyAll",
+
+  @doc("Allow traffic within the same namespace.")
+  AllowSameNamespace: "AllowSameNamespace",
+
+  string,
+}
+
 @added(Versions.v2025_03_02_naar)
-model FleetManagedNamespaceProfile {
-  @doc("Action if Kubernetes managed namespace with the same name already exists.")
-  adoptionPolicy?: AdoptionPolicy;
+@doc("The orchestrator profile to manage the lifecycle of the namespace.")
+model OrchestratorProfile {
```

Comment:
Orchestrator describes who will do the job (which customers care less about as it is implementation details), rather than how the job will be done.

I would suggest using:

```
PropagationPolicy {
  type: Placement | Direct
  PlacementProfile | DirectProfile
}
```